main_app/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, \
+                                        PermissionsMixin
+class Park(models.Model):
+    """Makes park instance for favorited park"""
+    name = models.CharField(max_length=300)
+    formatted_address = models.CharField(max_length=300)
+    opening_hours = models.CharField(max_length=1000, default='')
+    photo = models.CharField(max_length=1000, default='')
+    rating = models.CharField(max_length=100, default='0')
```

Comment:
Line 9 and 10 had `models.CharField` which is why they weren't working. They needed arguments `()`

Reply:
thank you for clarifying

Reply:
Thanks Travis! I really had some struggle with that part of it.